src/tests/system/tests/test_access_control_simple.py

```diff
@@ -0,0 +1,186 @@
+"""
+SSSD Authentication Test Cases
+
+:requirement: access control
+"""
+
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopologyGroup
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+@pytest.mark.importance("critical")
+def test_access_control__simple_filter_permits_user_login(client: Client, provider: GenericProvider):
+    """
+    :title: Simple access filter permits user login
+    :setup:
+        1. Create users ‘user1’ and ‘user2’
+        2. Configure SSSD with ‘access_provider = simple’, ‘simple_allow_users = user1’
+        3. Start SSSD
+    :steps:
+        1. Try to login as ‘user1’
+        2. Try to login as ‘user2’
+    :expectedresults:
+        1. User1 can login
+        2. User2 cannot login
+    :customerscenario: False
+    """
+    provider.user("user1").add(password="Secret123")
```

Comment:
You don't have to specify password="Secret123", this is the default value.  ```provider.user("user1").add()``` is the same

Reply:
Ahhhh I didn't know about that! Thanks Dan :)